client/client.go

```diff
@@ -179,6 +179,8 @@ type client struct {
 	tsDeadline sync.Map // Same as map[string]chan deadline
 	// dc-location -> *lastTSO
 	lastTSMap sync.Map // Same as map[string]*lastTSO
+
+	checkTSDeadlineCh chan struct{}
```

Comment:
It seems more reasonable to use `context`.

Reply:
why?

Reply:
May I ask why? The channel is more easy to use in this case.

Reply:
Sorry, I read something wrong.